roles/nova-data/tasks/main.yml

```diff
@@ -27,6 +27,17 @@
 
 - modprobe: name=nbd state=present
 
+- name: Turning off SMT mode for ppc64le
+  command: "ppc64_cpu --smt=off"
+  when: ansible_architecture == "ppc64le"
```

Comment:
By default, POWER8 system boot with SMT8 enabled.  Needs to be turned off in order to provision Vms, else KVM start can fail with “device or resource busy” message. So it has been disabled at boot level by adding this in rc.local.